AdsLib/AdsDef.cpp

```diff
@@ -78,18 +126,14 @@ AmsNetId::AmsNetId(const std::string& addr)
     }
 }
 
-bool AmsNetId::operator<(const AmsNetId& rhs) const
+bool AmsNetId::isValid() const
 {
-    for (unsigned int i = 0; i < sizeof(rhs.b); ++i) {
-        if (b[i] != rhs.b[i]) {
-            return b[i] < rhs.b[i];
-        }
-    }
-    return false;
+	static const AmsNetId empty {};

```

Comment:
Mixed usage of tab and spaces for indentation. Please replace all instances of tabs (there are other lines too) with 4 spaces.

Reply:
We have an [uncrustify](https://github.com/uncrustify/uncrustify) config for this:
https://github.com/Beckhoff/ADS/blob/master/tools/uncrustify.cfg
`make uncrustify`